lib/dcell/registries/mongodb_adapter.rb

```diff
@@ -1,8 +1,12 @@
 require 'mongoid'
+require 'bson'
 
 module DCell
   module Registry
     class MongodbAdapter
```

Comment:
For what it's worth, I kind of question the value of having this adapter at all. Mongo has a long history of correctness problems and isn't really reliable as a source of distributed consensus. I guess it might be convenient to certain people, but my guess is it will be buggy if anyone were to write anything real with DCell.